src/config.rs

```diff
@@ -674,6 +674,8 @@ create_config! {
     error_on_line_overflow: bool, true, false, "Error if unable to get all lines within max_width";
     error_on_line_overflow_comments: bool, true, false,
         "Error if unable to get comments within max_width";
+    error_on_line_overflow_strings: bool, true, false,
+        "Error if unable to get string letrais within max_width";
```

Comment:
`letrais` does not match any dictionary word.

Did you mean one of:

- `letters`
- `literal`
- `laterals`
- `literals`
- `lateral`

?

(Sorry, just having fun. Those are legitimate suggestions from a spell checker, though. You meant `literals` here.)

Reply:
Thank you, this will make my typo of the year for 2017.